tests/wpt/web-platform-tests/cssom/CSSStyleSheet.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>CSSOM - CSSStyleSheet interface</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <style>
+        body { width: 50%; }
+        #foo { height: 100px; }
+    </style>
+
+    <script>
+    test(function () {
+        var styleSheet = document.styleSheets[0];
```

Comment:
`document.getElementById("my-stylesheet")` returns CSSStyleDeclaration and `document.styleSheets[0]` returns CSSStyleSheet.
`document.getElementById("my-stylesheet").style` returns this style element's associated style rules I guess.